### DISCUSSION TITLE
如何让 Spring Security 「少管闲事」 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#202] [Announcements]
https://mazhuang.org/2021/12/26/spring-security/ 

 记两种让 Spring Security「少干点活」的方法。

**Comment by gcnyin** [comment]
博主在写spring?要不来搞go,rust（逃

**Comment by mzlogin** [comment]
> @gcnyin
> 博主在写spring?要不来搞go,rust（逃

哈哈哈哈兄台在哪里高就？页面右上角二维码可加微交流
